context/app/static/js/pages/Publication/Publication.jsx

```diff
@@ -42,7 +43,7 @@ function Publication({ publication, vignette_json }) {
   };
```

Comment:
GitHub's not letting me select it, but the `files: true` on line 41 here which makes the `files` header/section display no matter what should be swapped for `Boolean(files?.length)` to ensure the empty header is not displayed if there are no files to display in the `FileBrowser`. This is the reason for one of the failing tests.